src/components/head.js

```diff
@@ -23,10 +24,14 @@ export default (props) => {
     .filter(Boolean)
     .join(" | ")
   const description =
-    (frontmatter && frontmatter.description) || props.description
+    (frontmatter && frontmatter.description) || props.description || "Swizec helps you become a better frontend engineer with books, articles, talks, and workshops";
 
-  const image = props.image || getSocialCard(frontmatter)
-  const url = `${props.path}`
+  const socialImage = getSocialCard(frontmatter);
+  const image = `https://swizec.com${socialImage || defaultHero}`;
```

Comment:
Does this have any implications for how we end up hosting?

Reply:
What implications are you thinking about?
Anyway I tried it with Netlify and with Gatsby Cloud and it's working

Reply:
Just in terms of URL structure we’re allowed to use. Should be fine if you tested it :)